Simplified/Book/UI/NYPLBookCellDelegate+AudiobookBookmark.swift

```diff
@@ -23,7 +23,7 @@ import UIKit
       drmDeviceID: NYPLUserAccount.sharedAccount().deviceID,
       bookRegistryProvider: audiobookRegistryProvider,
       syncPermission: AccountsManager.shared.syncPermissionGranted,
-      annotationsSynchronizer: NYPLAnnotations.self)
+      annotationsSynchronizer: NYPLRootTabBarController.shared().annotationsSynchronizer)
```

Comment:
this is not ideal. I did this because (1) the NYPLAnnotations type is not objc-compatible, and (2) because this is in a class external to the reader.